packages/components/src/ResourceList/Resource/Resource.scss

```diff
@@ -1,10 +1,16 @@
+.center {
```

Comment:
Please put this under `.resource-item` to avoid errors using `:global` (if any, someday)

Reply:
we have naming isolation. it is very necessary ?

Reply:
no just a nice to have to avoid errors if one day someone use:
```scss
:global(.center) { /* something else */ }
```